Makefile

```diff
@@ -148,28 +148,43 @@ else
 OPM = $(shell which opm)
 endif
 endif
+
+.PHONY: osdk ## Download operator-sdk to /tmp/operator-sdk if it is not present on $PATH
+OSDK = /tmp/operator-sdk
+osdk:
+ifeq (,$(shell which operator-sdk 2>/dev/null))
+	@{ \
+	set -e ;\
+	wget --no-verbose -O $(OSDK) https://github.com/operator-framework/operator-sdk/releases/download/v1.15.0/operator-sdk_linux_amd64 -o operator-sdk;\
```

Comment:
We continue to download the SDK and oc in CI. We should consider injecting them into the image. Ref: [WINC-520](https://issues.redhat.com/browse/WINC-520).

You could say this is outside the scope of the PR, however you are now introducing yet another spot where we are downloading the SDK.